samples/sensor/lps22hh_i3c/prj.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_CBPRINTF_FP_SUPPORT=y
+
+CONFIG_SENSOR=y
+CONFIG_LPS22HH=y
```

Comment:
remove `CONFIG_LPS22HH` shouldn't be needed.

Reply:
removed.